primitives/src/vid/advz.rs

```diff
@@ -380,45 +294,48 @@ where
             .expect("num_storage_nodes log base arity should fit into usize");
         let height = height + 1; // avoid fully qualified syntax for try_into()
         let all_evals_commit = V::from_elems(height, &all_storage_node_evals).map_err(vid)?;
+        end_timer!(all_evals_commit_timer);
 
+        let common_timer = start_timer!(|| format!("compute {} KZG commitments", polys.len()));
         let common = Common {
             poly_commits: polys
                 .iter()
                 .map(|poly| P::commit(&self.ck, poly))
                 .collect::<Result<_, _>>()
                 .map_err(vid)?,
             all_evals_digest: all_evals_commit.commitment().digest(),
-            elems_len,
-        };
-
-        let commit = {
-            let mut hasher = H::new();
-            for poly_commit in common.poly_commits.iter() {
-                // TODO compiler bug? `as` should not be needed here!
-                (poly_commit as &P::Commitment)
-                    .serialize_uncompressed(&mut hasher)
-                    .map_err(vid)?;
-            }
-            hasher.finalize()
+            bytes_len: payload_len,
         };
+        end_timer!(common_timer);
 
-        let pseudorandom_scalar = Self::pseudorandom_scalar(&common)?;
+        let misc_timer = start_timer!(|| {
```

Comment:
We can omit this timer.

Reply:
done in https://github.com/EspressoSystems/jellyfish/pull/389/commits/a0678336d0061f4f6fb2406401a5cc2179191c2e